src/plugins/imagery/components/ImageryViewLayout.vue

```diff
@@ -66,8 +73,8 @@
          :class="{'is-paused': isPaused}"
          @scroll="handleScroll"
     >
-        <div v-for="(imageData, index) in imageHistory"
-             :key="index"
+        <div v-for="(imageData, i) in imageHistory"
+             :key="i"
```

Comment:
We cannot use the index as a key in `v-for`. This code predates this change, so I will file a followup.

Reply:
Updated!